README.md

```diff
@@ -63,6 +63,17 @@ Please report any issues or bug reports on the [GitHub Issues](https://github.co
 This module is licensed under the **The MIT License**. Please see the [LICENSE.txt](LICENSE.txt) file for details.
 
 ## Releases
+## v0.4.3 (2019-09-xx)
+- French translation (@brunoduc)
+- Various translations fixes (@vlmaksime, @dagwieers)
+- Ensure Kodi 19 compatibility (@mediaminister)
+- Configurable temporary download directory for devices with limited space (@horstle)
+- Configurable Widevine CDM update frequency (@horstle)
+- Fix add-on settings crashes when InputStream Adaptive is missing (@mediaminister)
+- Improve unicode character support (@mediaminister)
+- Improve testing framework (@dagwieers)
+- Improve code structure (@dagwieers)
```

Comment:
As I mentioned before, I am not sure if these two belong in the changelog.
It is very generic, and something we will be doing as part of changes anyway.
So I hate to see this being part of the changelog in every new release (see previous notes).

Reply:
Removed.